### ISSUE TITLE
[CI] Create GitHub workflow for building site and publishing site

### ISSUE THREAD

**Issue by leecalcote** [layer5io/layer5-ng#3]
Potentially use this or another GitHub action - https://github.com/marketplace/actions/gatsby-publish

**Comment by MukulMaheshwari** [comment]
Hey @leecalcote I would like to work on this. I have worked with github actions before in a open source project.

**Comment by leecalcote** [comment]
@MukulMaheshwari that sounds great. Yes, please do jump in here. You're help is much appreciated. 👍

**Comment by leecalcote** [comment]
@MukulMaheshwari I invite you to join the Layer5 Slack workspace as well - http://slack.layer5.io. There's a warm and welcoming collection of contributors in there, who can both provide more context about this repository and the other initiatives the community is stewarding. There are a number of resources linked here as well - https://meshery.layer5.io/docs/project/contributing

**Comment by MukulMaheshwari** [comment]
Yeah sure , I have joined there & would like to know how to build it on local machine.

**Comment by leecalcote** [comment]
@MukulMaheshwari fyi - Netlify deployment previews now working (#12)

**Comment by leecalcote** [comment]
@MukulMaheshwari fyi - https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords

**Comment by leecalcote** [comment]
Reopening to ensure that the build process in fact deploys the site.
